spec/services/authorization_service_spec.rb

```diff
@@ -20,6 +20,14 @@
       expect(user.remember_token).to be_truthy
       expect(user.remember_created_at).to be_truthy
     end
+
+    it "queues a slack message to be sent for a user whose identity is brand new" do
+      service.auth.extra.raw_info.created_at = 1.minute.ago.rfc3339
```

Comment:
Why do we need to use `rfc3339` here?

Reply:
Because that's how that value is defined in the preparation data, it mimics what Github sends during login:

https://github.com/thepracticaldev/dev.to/blob/bc8c12649720adc90c9fc21a632ff1eb03121c3c/spec/support/omniauth_macros.rb#L29